### PR TITLE
Update Etcd to 3.4.3

### PR DESCRIPTION
**Pull request by Sindica**

### CODE REVIEW

File: cluster/gce/manifests/etcd.manifest [hunk 1]
```diff
@@ -23,14 +23,14 @@
...
+              "if [ -e /usr/local/bin/migrate-if-needed.sh ]; then /usr/local/bin/migrate-if-needed.sh 1>>/var/log/etcd{{ suffix }}.log 2>&1; fi; exec /usr/local/bin/etcd --name etcd-{{ hostname }} --listen-peer-urls {{ etcd_protocol }}://{{ host_ip }}:{{ server_port }} --initial-advertise-peer-urls {{ etcd_protocol }}://{{ hostname }}:{{ server_port }} --advertise-client-urls {{ etcd_apiserver_protocol }}://127.0.0.1:{{ port }} --listen-client-urls {{ etcd_apiserver_protocol }}://{{ listen_client_ip }}:{{ port }} {{ quota_bytes }} --data-dir /var/etcd/data{{ suffix }} --initial-cluster-state {{ cluster_state }} --initial-cluster {{ etcd_cluster }} {{ etcd_creds }} {{ etcd_apiserver_creds }} {{ etcd_extra_args }} 1>>/var/log/etcd{{ suffix }}.log 2>&1"
```
Comment (yb01) [inline cluster/gce/manifests/etcd.manifest:26]: any reason this parameter is removed ? cluster-id {{ etcd_cluster_id }}

File: hack/lib/etcd.sh [hunk 2]
```diff
@@ -100,8 +100,8 @@ kube::etcd::start() {
...
-  etcd --name ${ETCD_NAME} --cluster-id ${ETCD_CLUSTER_ID} --listen-peer-urls "http://${ETCD_HOST}:${ETCD_PEER_PORT}" --advertise-client-urls "${KUBE_INTEGRATION_ETCD_URL}" --data-dir "${ETCD_DIR}" --listen-client-urls "${KUBE_INTEGRATION_ETCD_URL},http://${ETCD_LOCAL_HOST}:${ETCD_PORT}" --initial-advertise-peer-urls "http://${ETCD_HOST}:${ETCD_PEER_PORT}" --initial-cluster-token "etcd-cluster-1" --initial-cluster "${ETCD_NAME}=http://${ETCD_HOST}:${ETCD_PEER_PORT}" --initial-cluster-state "new"  --debug 2> "${ETCD_LOGFILE}" >/dev/null &
+  kube::log::info "etcd --name ${ETCD_NAME} --listen-peer-urls http://${ETCD_HOST}:${ETCD_PEER_PORT}   --advertise-client-urls ${KUBE_INTEGRATION_ETCD_URL} --data-dir ${ETCD_DIR} --listen-client-urls ${KUBE_INTEGRATION_ETCD_URL},http://${ETCD_LOCAL_HOST}:${ETCD_PORT} --listen-peer-urls http://${ETCD_HOST}:${ETCD_PEER_PORT} --debug > \"${ETCD_LOGFILE}\" 2>/dev/null"
```
Comment (yb01) [inline hack/lib/etcd.sh:103]: same here the cluster-id parameter is removed.

File: cluster/gce/manifests/etcd.manifest (hunk 1 above)
Comment (Sindica) [reply cluster/gce/manifests/etcd.manifest:26]: cluster-id was introduced for customized ETCD

**Review by yb01 — APPROVED** [review]
/lgtm

**Pull request merged**